Tasks/Common/azure-arm-rest/azure-app-service.ts

```diff
@@ -0,0 +1,185 @@
+import msRestAzure = require('./azure-arm-common');
+import webClient = require('./webClient');
+import azureServiceClient = require('./AzureServiceClient');
+import tl = require('vsts-task-lib/task');
+import Q = require('q');
+import * as querystring from "querystring";
+var parseString = require('xml2js').parseString;
+
+export class AzureAppService {
+    private appServiceName: string;
+    private resourceGroupName: string;
+    private slotName: string;
+    private endpoint: any;
```

Comment:
is it possible to include an interface for endpoint as part of this PR ?

Reply:
yes. working on the same